packages/perseus/src/widgets/interactive-graphs/graphs/use-transform.ts

```diff
@@ -1,10 +1,36 @@
-import {useTransformContext, vec} from "mafs";
-
-export const useTransform = (...points: vec.Vector2[]) => {
-    const {viewTransform, userTransform} = useTransformContext();
-    const transformToPx = vec.matrixMult(viewTransform, userTransform);
-    const transformedPoints = points.map((point) =>
-        vec.transform(point, transformToPx),
-    );
-    return transformedPoints;
+import {type Interval, vec} from "mafs";
+
+import useGraphConfig from "../reducer/use-graph-config";
+
+const matrixBuilder = vec.matrixBuilder;
+
+export type GraphDimensions = {
+    range: [Interval, Interval];
+    width: number; // pixels
+    height: number; // pixels
+};
+
+export function vectorToPixel(
+    points: vec.Vector2[],
```

Comment:
Related to Ben's comment below.  Maybe we name this `vectors` to enforce the point that we're dealing with vectors (distances) and not x,y locations on the graph?